apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -148,10 +157,20 @@ const AssignContentTypeSection = (props: Props) => {
     <Stack spacing="spacingL" flexDirection="column" alignItems="flex-start">
       <Box>
         <Subheading marginBottom="spacingXs">Content type configuration</Subheading>
+        <Paragraph>
+          Configure content types below that are connected to pages on your website where you're
+          tracking Google Analytics data. You'll need to specify the "slug" field used to generate
+          the page path in your website's URL, and optionally a "prefix" if one exists in front of
+          the URL page path.
+        </Paragraph>
         <Paragraph marginBottom="none">
-          Select which content types will show the Google Analytics functionality in the sidebar.
-          Specify the slug field that is used for URL generation in your application. Optionally,
-          specify a prefix for the slug.
+          <TextLink
+            href="https://support.google.com/analytics/answer/10759417"
+            target="_blank"
+            icon={<ExternalLinkIcon />}
+            alignIcon="end">
+            WRONG See our help documentation for more details
```

Comment:
What's going on here with all caps yelling at the user? 😂

Reply:
> Add link to docks. It says "WRONG" because we don't have the real link and this intended to stick out like a sore thumb.

Very much intentional because this is the wrong link and it's super easy to overlook forgetting it. Should have it in the next day or so.